plugins/GSdx/GSdx.rc

```diff
@@ -138,8 +138,6 @@ BEGIN
     CONTROL         "Wild Arms Offset",IDC_WILDHACK,"Button",BS_AUTOCHECKBOX | WS_TABSTOP,108,84,70,8
     // Columns end
     //
-    RTEXT           "MSAA:",IDC_MSAA_TEXT,54,99,22,8
-    COMBOBOX        IDC_MSAACB,80,96,116,63,CBS_DROPDOWNLIST | WS_VSCROLL | WS_TABSTOP
```

Comment:
The options might need to be moved one slot as well as the height adjusted to compensate for the removed option, it can be done at a later commit/pr tho.